api/src/main/java/com/datastrato/gravitino/authorization/AuthorizationRole.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2024 Datastrato Pvt Ltd.
+ * This software is licensed under the Apache License version 2.
+ */
+package com.datastrato.gravitino.authorization;
+
+/** Authorization Role interface. */
```

Comment:
Can you please add more comments about the usage of this interface and how developers use this interface?

Reply:
Added some comments on these classes and functions.
When I finish developing Ranger authorization, I will add a development document.